components/engine_rocks/src/logger.rs

```diff
@@ -8,13 +8,13 @@ pub struct RocksdbLogger;
 impl Logger for RocksdbLogger {
     fn logv(&self, log_level: InfoLogLevel, log: &str) {
         match log_level {
-            InfoLogLevel::Debug => debug!(#"rocksdb_log","{}", log),
-            InfoLogLevel::Info | InfoLogLevel::Header | InfoLogLevel::NumInfoLog => {
-                info!(#"rocksdb_log","{}", log)
-            }
-            InfoLogLevel::Warn => warn!(#"rocksdb_log","{}", log),
-            InfoLogLevel::Error => error!(#"rocksdb_log","{}", log),
-            InfoLogLevel::Fatal => crit!(#"rocksdb_log","{}", log),
+            InfoLogLevel::Header => info!(#"rocksdb_log_header", "{}", log),
+            InfoLogLevel::Debug => debug!(#"rocksdb_log", "{}", log),
+            InfoLogLevel::Info => info!(#"rocksdb_log", "{}", log),
+            InfoLogLevel::Warn => warn!(#"rocksdb_log", "{}", log),
+            InfoLogLevel::Error => error!(#"rocksdb_log", "{}", log),
+            InfoLogLevel::Fatal => crit!(#"rocksdb_log", "{}", log),
+            _ => {}
```

Comment:
```suggestion
            InfoLogLevel::NumInfoLog => {}
```
actually, I don't know what it is used for

Reply:
The `NUM_INFO_LOG_LEVELS` is a c/c++ trick to get max enum number. In rust we do not need them. Let me update rust-rocksdb to remove this enum.

https://github.com/tikv/rocksdb/blob/6.4.tikv/include/rocksdb/env.h#L968